import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -0,0 +1,329 @@
+	policyName := base.GenerateRandomString() + "Policy"
+
+	output, _ := exportThrottlePolicy(t, policyName, args.SrcAPIM.GetEnvName(), args.Type)
+	assert.Contains(t, output, "Error exporting Throttling Policies", "Exportation error expected")
+}
+
+func createExportedThrottlePolicyFile(t *testing.T, client *apim.Client, policyType string, corrupted bool) (string, interface{}, error) {
+
+	t.Helper()
+	var exportedPolicy utils.ExportThrottlePolicy
+	exportedPolicy.Type = "throttling policy"
+
+	if !corrupted {
+		switch policyType {
+		case apim.SubscriptionThrottlePolicyType:
+			exportedPolicy.Subtype = subscriptionPolicySubType
+		case apim.AdvancedThrottlePolicyType:
+			exportedPolicy.Subtype = advancedPolicySubType
+		case apim.CustomThrottlePolicyType:
+			exportedPolicy.Subtype = customPolicySubType
+		case apim.ApplicationThrottlePolicyType:
+			exportedPolicy.Subtype = applicationPolicySubType
+		}
+	}
+
+	exportedPolicy.Version = "v4.1.0"
+	policyData := client.GenerateSampleThrottlePolicyData(policyType)
+	policyMap, _ := PolicyStructToMap(policyData)
+	var yamlMap yaml.MapSlice
+	yamlBytes, err := yaml.Marshal(policyMap)
+	err = yaml.Unmarshal(yamlBytes, &yamlMap)
+	if err != nil {
+		return "", policyData, err
+	}
+	exportedPolicy.Data = yamlMap
+	policyMarshaledData, _ := yaml.Marshal(exportedPolicy)
+
+	tmpDir, err := ioutil.TempDir("", "apim")
+	if err != nil {
+		_ = os.RemoveAll(tmpDir)
+		return "", policyData, err
+	}
+	tempFilePath := tmpDir + "/temp.yaml"
+	err = ioutil.WriteFile(tempFilePath, policyMarshaledData, os.ModePerm)
+	if err != nil {
+	}
+	return tempFilePath, policyData, err
+}
+
+// Adds a new Throttling Policy to an env
+func AddNewThrottlePolicy(t *testing.T, client *apim.Client, username, password, policyType string) interface{} {
+	client.Login(username, password)
+	generatedPolicy := client.GenerateSampleThrottlePolicyData(policyType)
+	addedPolicy := client.AddThrottlePolicy(t, generatedPolicy, policyType)
+	return addedPolicy
+}
+
+// Exports Throttling Policy from an env
+func exportThrottlePolicy(t *testing.T, name, env, policyTypeFlagValue string) (string, error) {
+	var output string
+	var err error
+	if policyTypeFlagValue != "" {
+		output, err = base.Execute(t, "export", "policy", "rate-limiting", "-n", name, "-t", policyTypeFlagValue, "-e", env, "-k", "--verbose")
+	} else {
+		output, err = base.Execute(t, "export", "policy", "rate-limiting", "-n", name, "-e", env, "-k", "--verbose")
+	}
+	return output, err
+}
+
+// Imports Throttling policy to an env
+func importThrottlePolicy(t *testing.T, args *PolicyImportExportTestArgs) (string, error) {
+	var output string
+	var err error
+	if args.Update {
+		output, err = base.Execute(t, "import", "policy", "rate-limiting", "-e", args.DestAPIM.GetEnvName(), "-f", args.ImportFilePath, "-u")
+	} else {
+		output, err = base.Execute(t, "import", "policy", "rate-limiting", "-e", args.DestAPIM.GetEnvName(), "-f", args.ImportFilePath)
+	}
+	return output, err
```

Comment:
Need to add t.cleanUp here as well. Refer [1].

[1] https://github.com/wso2/product-apim-tooling/blob/5fce77894c0117a8e45239e3f6abaa36034d8248/import-export-cli/integration/testutils/api_testUtils.go#L372